skia-bindings/build_support/binaries_config.rs

```diff
@@ -146,7 +154,41 @@ impl BinariesConfiguration {
 
         let target = cargo::target();
 
-        cargo::add_static_link_libs(&target, self.built_libraries());
+        cargo::add_static_link_libs(&target, self.built_libraries(true));
         cargo::add_link_libs(&self.link_libraries);
     }
+
+    /// Import library and additional files from `from_dir` to the output directory.
+    pub fn import(&self, from_dir: &Path, import_bindings_libraries: bool) -> io::Result<()> {
+        let output_directory = &self.output_directory;
+        self.copy_libs_and_additional_files(from_dir, output_directory, import_bindings_libraries)
+    }
+
+    /// Export library and additional files from the output directory to a `to_dir`.
+    pub fn export(&self, to_dir: &Path) -> io::Result<()> {
+        let output_directory = &self.output_directory;
+        self.copy_libs_and_additional_files(output_directory, to_dir, true)
+    }
+
+    fn copy_libs_and_additional_files(
+        &self,
+        from_dir: &Path,
+        to_dir: &Path,
+        copy_bindings_libraries: bool,
+    ) -> io::Result<()> {
+        fs::create_dir_all(&to_dir)?;
+
+        let target = cargo::target();
+
+        for lib in self.built_libraries(copy_bindings_libraries) {
+            let filename = &target.library_to_filename(lib);
+            fs::copy(from_dir.join(filename), to_dir.join(filename))?;
```

Comment:
There’s a lot of copying which might be slow and use disk space - I wonder if it’s better to symlink if available

(ie. `std::os::unix::fs::symlink` or `std::os::windows::fs::symlink_file`)

Reply:
I suggested to take care of that at a later time if this gets to be an issue in some constrained build environments.

Reply:
Sounds good to me